lib/cog/commands/role.ex

```diff
@@ -25,16 +23,16 @@ defmodule Cog.Commands.Role do
   option "revoke", type: "bool"
   option "list", type: "bool"
 
-  option "user", type: "string"
   option "group", type: "string"
 
   permission "manage_roles"
-  permission "manage_users"
   permission "manage_groups"
 
-  rule "when command is #{Cog.embedded_bundle}:role must have #{Cog.embedded_bundle}:manage_roles"
-  rule "when command is #{Cog.embedded_bundle}:role must have #{Cog.embedded_bundle}:manage_users"
-  rule "when command is #{Cog.embedded_bundle}:role must have #{Cog.embedded_bundle}:manage_groups"
+  rule "when command is #{Cog.embedded_bundle}:role with option[create] == true must have #{Cog.embedded_bundle}:manage_roles"
+  rule "when command is #{Cog.embedded_bundle}:role with option[drop] == true must have #{Cog.embedded_bundle}:manage_roles"
+  rule "when command is #{Cog.embedded_bundle}:role with option[list] == true must have #{Cog.embedded_bundle}:manage_roles"
+  rule "when command is #{Cog.embedded_bundle}:role with option[grant] == true must have #{Cog.embedded_bundle}:manage_groups"
```

Comment:
Same as above, shouldn't we require both manage_groups and manage_roles perms here?